src/schemas/authorization.rs

```diff
@@ -57,14 +70,23 @@ impl TryInto<Request> for AuthorizationCall {
             Ok(r) => r,
             Err(e) => return Err(e.into()),
         };
+        let entities = match self.entities {            
+            Some(et) => match Entities::from_json_value(et, None) {
+                Ok(et) => {
+                    et
+                },
+                Err(e) => return Err(e.into()),
+            },
+            None => Entities::empty(),
```

Comment:
Entities::empty() returns empty entities instead of None, in case no Entities was provided in the request you should pass None instead of empty Entities